get-started/overview.md

```diff
@@ -174,29 +173,28 @@ the default registry configuration):
     directories in its local filesystem.
 
 4.  Docker creates a network interface to connect the container to the default
-    network, since you did not specify any networking options. This includes
+    network, since you didn't specify any networking options. This includes
     assigning an IP address to the container. By default, containers can
     connect to external networks using the host machine's network connection.
 
 5.  Docker starts the container and executes `/bin/bash`. Because the container
     is running interactively and attached to your terminal (due to the `-i` and `-t`
-    flags), you can provide input using your keyboard while the output is logged to
+    flags), you can provide input using your keyboard while Docker logs the output to
     your terminal.
 
-6.  When you type `exit` to terminate the `/bin/bash` command, the container
-    stops but is not removed. You can start it again or remove it.
+6.  When you run `exit` to terminate the `/bin/bash` command, the container
+    stops but isn't removed. You can start it again or remove it.
 
 ## The underlying technology
 Docker is written in the [Go programming language](https://golang.org/) and takes
 advantage of several features of the Linux kernel to deliver its functionality.
 Docker uses a technology called `namespaces` to provide the isolated workspace
-called the *container*. When you run a container, Docker creates a set of
-*namespaces* for that container.
+called the container. When you run a container, Docker creates a set of
+namespaces for that container.
 
 These namespaces provide a layer of isolation. Each aspect of a container runs
 in a separate namespace and its access is limited to that namespace.
 
 ## Next steps
```

Comment:
Is there a newline after the heading? hard to tell from the GH review UI but it looks like we need to add one

```suggestion
## Next steps

```